packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -67,6 +68,10 @@ afterEach( () => {
 	mockFetchRichUrlData?.mockReset(); // Conditionally reset as it may NOT be a mock.
 } );
 
+function render( ui, options ) {
+	return baseRender( ui, { wrapper: SlotFillProvider, ...options } );
```

Comment:
Can't we remove `options` altogether?

```suggestion
function render( ui ) {
	return baseRender( ui, { wrapper: SlotFillProvider } );
```

Reply:
Yes, `options` can be removed and I did that in the latest version of the PR.